guide/listing-status.md

```diff
@@ -27,6 +27,7 @@ with the following in the JSON body:
 | `status`  | `StandardStatus` | A standardized status that enables Zentail to take particular actions. See the models for the most up to date set of values |
 | `channelStatus` | `string` | A free-form status that reflects the status of the listing in your terms |
 | `channelId` | `string` | A unique identifier for this listing on your platform |
+| `buyable` | `boolean` | Set to true if the listing is published on your channel and also has available inventory. This flag indicates to Zentail sellers that the product is live and sales should be expected. |
```

Comment:
I didn't get a chance to read the comment on the last PR but is this true? I thought a listing could be suppressed but buyable and that's why we made this a flag instead of a new status. should it say "Set to true if the listing is able to be purchased (this can only be true if the status is `PUBLISHED` or `SUPPRESSED`)"

Reply:
fixed